src/executable-code/index.js

```diff
@@ -177,14 +177,12 @@ export default class ExecutableCode {
 
   getTheme(targetNode) {
     const theme = targetNode.getAttribute(ATTRIBUTES.THEME);
-    switch (theme) {
-      case THEMES.DARCULA:
-        return THEMES.DARCULA;
-      case THEMES.IDEA:
-        return THEMES.IDEA;
-      default:
-        return THEMES.DEFAULT;
+
+    if (theme && theme !== THEMES.DARCULA && theme !== THEMES.IDEA) {
+      console.warn(`Custom theme '${theme}' requires custom css by developer, you might use default values for reduce size – ${THEMES.DARCULA} or ${THEMES.IDEA}.`);
```

Comment:
@AlexanderPrendota, warning for better understanding misspells, text may be better :)

Reply:
#copyediting ? =)

Reply:
Is it suggestion or question? :) It my pre-intermediate level, do you have better text? ;) Or is it good enough and i can merge?